pkg/hypervisor/hypervisor.go

```diff
@@ -1,4 +1,4 @@
-// Package hypervisor implements management node
+// Package hypervisor implements management visor
```

Comment:
It should be `management node`, because `node` is not related to `visor` here.

Reply:
true